src/main/java/com/snack/news/exception/base/NotFoundException.java

```diff
@@ -1,16 +1,12 @@
 package com.snack.news.exception.base;
 
+import lombok.Getter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused import - lombok.Getter.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)